lib/rules/selector-attribute-brackets-space-inside/index.js

```diff
@@ -101,14 +100,18 @@ function rule(expectation, options, context) {
 				});
 			});
 
-			if (hasFixed) {
+			if (hasFixed && fixedSelector) {
```

Comment:
[note] `fixedSelector` is a `string` or `undefined`. See `lib/utils/parseSelector.js`.